Library/Formula/knot.rb

```diff
@@ -30,6 +30,7 @@ def install
     system "autoreconf", "-i", "-f" if build.head?
     system "./configure", "--disable-debug",
                           "--disable-dependency-tracking",
+                          "--disable-fastparser",
```

Comment:
Can we add a Ruby comment somewhere so this doesn't get left here forever? Maybe link to the upstream bug report and explanation of why it's needed? Thanks!